pyroSAR/drivers.py

```diff
@@ -1751,19 +1871,37 @@ def encode(string, encoding='utf-8'):
         else:
             return string
     
-    def export2shp(self, shp):
+    def export2shp(self, path):
         """
         export the database to a shapefile
 
         Parameters
         ----------
-        shp: str
-            the name of the shapefile to be written
-
+        path: str
+            the path of the folder for the shapefile of data to be written
+            this will overwrite other files with the same name.
+            If only a folder is given in path (path ends with '/'),
+            the default name of the shapefile is 'pyroSAR_data.shp'.
+            Otherwise '.shp' is appended to the path, if missing.
+            
         Returns
         -------
         """
-        ogr2ogr(self.dbfile, shp, options={'format': 'ESRI Shapefile'})
+        if path.endswith('/'):
```

Comment:
Could we use checks from os.path for being a directory?
I believe, that how it is done now would fail in windows.